packs/data/feats.db/skill-mastery-rogue.json

```diff
@@ -33,7 +33,340 @@
+                    {
+                        "label": "PF2E.SkillNat",
+                        "predicate": {
+                            "all": [
+                                "skill:nat:rank:1"
+                            ]
+                        },
+                        "value": "nat"
+                    },
+                    {
+                        "label": "PF2E.SkillOcc",
+                        "predicate": {
+                            "all": [
+                                "skill:occ:rank:1"
+                            ]
+                        },
+                        "value": "occ"
+                    },
+                    {
+                        "label": "PF2E.SkillPrf",
+                        "predicate": {
+                            "all": [
+                                "skill:prf:rank:1"
+                            ]
+                        },
+                        "value": "prf"
+                    },
+                    {
+                        "label": "PF2E.SkillRel",
+                        "predicate": {
+                            "all": [
+                                "skill:rel:rank:1"
+                            ]
+                        },
+                        "value": "rel"
+                    },
+                    {
+                        "label": "PF2E.SkillSoc",
+                        "predicate": {
+                            "all": [
+                                "skill:soc:rank:1"
+                            ]
+                        },
+                        "value": "soc"
+                    },
+                    {
+                        "label": "PF2E.SkillSte",
+                        "predicate": {
+                            "all": [
+                                "skill:ste:rank:1"
+                            ]
+                        },
+                        "value": "ste"
+                    },
+                    {
+                        "label": "PF2E.SkillSur",
+                        "predicate": {
+                            "all": [
+                                "skill:sur:rank:1"
+                            ]
+                        },
+                        "value": "sur"
+                    },
+                    {
+                        "label": "PF2E.SkillThi",
+                        "predicate": {
+                            "all": [
+                                "skill:thi:rank:1"
+                            ]
+                        },
+                        "value": "thi"
+                    }
+                ],
+                "flag": "skillExpert",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.skillMastery.ExpertPrompt"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.skills.{item|flags.pf2e.rulesSelections.skillExpert}.rank",
```

Comment:
I've just tested this RE, and the skills don't increase (either trained to expert or expert to master). No errors in the console

Reply:
This was a bug in the system I think and should have been fixed.

Reply:
Confirmed it is fixed now